substrate/frame/contracts/src/benchmarking/mod.rs

```diff
@@ -16,7 +16,7 @@
 // limitations under the License.
 
 //! Benchmarks for the contracts pallet
-
+#![allow(unused)]
```

Comment:
Can we put those per case instead of deactivating for the whole module?

Reply:
Ah right that should not be needed that was just me experimenting with the new syntax